cicecore/cicedynB/general/ice_flux.F90

```diff
@@ -33,29 +33,31 @@ module ice_flux
 
       !-----------------------------------------------------------------
       ! Dynamics component
+      ! All variables are assumed to be on the atm or ocn thermodynamic
+      ! grid except as noted
       !-----------------------------------------------------------------
 
       real (kind=dbl_kind), dimension (:,:,:), allocatable, public :: &
 
        ! in from atmos (if .not.calc_strair)  
-         strax   , & ! wind stress components (N/m^2)
-         stray   , & ! 
+         strax   , & ! wind stress components (N/m^2), on grid_atm_dynu
+         stray   , & !                                 on grid_atm_dynv
 
        ! in from ocean
-         uocn    , & ! ocean current, x-direction (m/s)
-         vocn    , & ! ocean current, y-direction (m/s)
-         ss_tltx , & ! sea surface slope, x-direction (m/m)
-         ss_tlty , & ! sea surface slope, y-direction
+         uocn    , & ! ocean current, x-direction (m/s),     on grid_ocn_dynu
+         vocn    , & ! ocean current, y-direction (m/s),     on grid_ocn_dynv
+         ss_tltx , & ! sea surface slope, x-direction (m/m), on grid_ocn_dynu
+         ss_tlty , & ! sea surface slope, y-direction,       on grid_ocn_dynv
          hwater  , & ! water depth for seabed stress calc (landfast ice) 
 
        ! out to atmosphere
-         strairxT, & ! stress on ice by air, x-direction
-         strairyT, & ! stress on ice by air, y-direction
+         strairxT, & ! stress on ice by air, x-direction at T points, computed in icepack
+         strairyT, & ! stress on ice by air, y-direction at T points, computed in icepack
 
        ! out to ocean          T-cell (kg/m s^2)
        ! Note, CICE_IN_NEMO uses strocnx and strocny for coupling
-         strocnxT, & ! ice-ocean stress, x-direction
-         strocnyT    ! ice-ocean stress, y-direction
+         strocnxT, & ! ice-ocean stress at T points, x-direction at T points, mapped from strocnx, per ice fraction
+         strocnyT    ! ice-ocean stress at T points, y-direction at T points, mapped from strocny, per ice fraction
```

Comment:
Can take out the second "at T points" in these lines.  Also "per ice fraction" would normally be "per unit ice area"

Reply:
Fixed in upcoming PR.